test/t/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+set(PROVE_DEBUG_TEST_FILES 001_extension.pl)
+
+if(CMAKE_BUILD_TYPE MATCHES Debug)
+  list(APPEND PROVE_TEST_FILES ${PROVE_DEBUG_TEST_FILES})
+endif(CMAKE_BUILD_TYPE MATCHES Debug)
+
+foreach(P_FILE ${PROVE_TEST_FILES})
+  configure_file(${P_FILE} ${CMAKE_CURRENT_BINARY_DIR}/${P_FILE} COPYONLY)
+endforeach(P_FILE)
+
+set(MODULE_PATHNAME "$libdir/timescaledb-${PROJECT_VERSION_MOD}")
+configure_file(functions.sql.in functions.sql)
```

Comment:
Some test sql files use `:MODULE_PATHNAME`, because it is set at runtime via psql variables. The solution I use here is different and sets the `@MODULE_PATHNAME@` to replace it at configuration (cmake) time. This is also why I added the .in extension. I think both are valid approaches, but the runtime-option isn't easily available for TAP tests.

Also, this sql file isn't strictly a test, but rather a library file. But not sure that matters.